src/main/java/com/integration/githubintegration/model/RepoDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.integration.githubintegration.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class RepoDTO {
+    private int id;
+    private String name;
+    private String full_name;
```

Comment:
Dla przeciwczenia dodac login ownera repozytorium

Reply:
Zamiast robic to w taki sposob i łamać konwencje przyjeta (camelCase) powinnismy zmienna nazwac normalnie fullName a nad nia dac adnotacje @JsonProperty("full_name"). Adnotacja ta spowoduje ze w momencie gdy bedzie matchowane ktory parametr jest ktorym w jsonie to dla fullName wartosc bedzie brana spod nazwy w JSONie "full_name" - dziala dla procesu serializacji jak i deserializacji.
Serializacja -> obiekt to json
Deserializacja -> json to obiekt

Reply:
Dodaj dodtkowo login ownera z repo.